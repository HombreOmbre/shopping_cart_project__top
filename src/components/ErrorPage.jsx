import styles from "../styles/ErrorPage.module.scss";

export const ErrorPage = () => {
    return (
        <div className={styles.errorPage}>
            <h1 className={styles.header}>Ooops!</h1>
            <p className={styles.errorTxt}>Something gone wrong...</p>
        </div>
    )
}