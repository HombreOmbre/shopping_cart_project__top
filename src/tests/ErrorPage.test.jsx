import { expect, describe, it } from "vitest";
import { render, screen } from '@testing-library/react';
import { ErrorPage } from "../components/ErrorPage.jsx";

describe("ErrorPage", () => {
    it("renders without crashing", () => {
        render(<ErrorPage />);

        expect(screen.getByText("Ooops!")).toBeInTheDocument();
        expect(screen.getByText("Something gone wrong...")).toBeInTheDocument();
    });

    it("matches snapshot", () => {
        const { asFragment } = render(<ErrorPage />);

        expect(asFragment()).toMatchSnapshot();
    })
})
