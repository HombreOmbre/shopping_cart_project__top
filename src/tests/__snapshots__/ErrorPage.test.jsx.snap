// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ErrorPage > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="_errorPage_580988"
  >
    <h1
      class="_header_580988"
    >
      Ooops!
    </h1>
    <p
      class="_errorTxt_580988"
    >
      Something gone wrong...
    </p>
  </div>
</DocumentFragment>
`;
